tests/unit/test_character_specifications.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Should this file be renamed since no longer pertains to "character"?

Reply:
Not a bad idea, I'd like to address this in a follow-up that further consolidates porter entities and logic.